common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1079,6 +1079,12 @@ def get_items(self, course_locator, settings=None, content=None, qualifiers=None
                 For substring matching pass a regex object.
                 For split,
                 you can search by ``edited_by``, ``edited_on`` providing a function testing limits.
+            include_orphans (boolean): Optionally include course orphans in the list. Orphans are items which aren't
+                in the course tree.
+                True - This would return all items irrespective of course in tree checking. It may fetch orphans
+                if present in the course.
+                False/ or not set - if we want only those items which are in the course tree. This would ensure no
```

Comment:
This is just `False`. This function has the same behavior if `include_orphans` is True or not set.